integration/system/info_linux_test.go

```diff
@@ -43,3 +47,71 @@ func TestInfoAPIVersioned(t *testing.T) {
 	assert.Check(t, is.Contains(out, "ExecutionDriver"))
 	assert.Check(t, is.Contains(out, "not supported"))
 }
+
+// TestInfoDiscoveryBackend verifies that a daemon run with `--cluster-advertise` and
+// `--cluster-store` properly returns the backend's endpoint in info output.
+func TestInfoDiscoveryBackend(t *testing.T) {
+	skip.If(t, testEnv.IsRemoteDaemon, "cannot run daemon when remote daemon")
+
+	const (
+		discoveryBackend   = "consul://consuladdr:consulport/some/path"
+		discoveryAdvertise = "1.1.1.1:2375"
+	)
+
+	d := daemon.New(t)
+	d.Start(t, "-H=0.0.0.0:23756", "-H="+d.Sock(), "--cluster-store="+discoveryBackend, "--cluster-advertise="+discoveryAdvertise)
```

Comment:
Why `-H=0.0.0.0:23756` ?

Reply:
Good catch; copy/pasta; removing

Reply:
It's gone.. let's pretend it was never there 😂